lua/entities/gmod_wire_keyboard/remap.lua

```diff
@@ -230,6 +230,100 @@ remap[KEY_I] = "í"
 remap[KEY_O] = "ó"
 remap[KEY_U] = "ú"
 
+----------------------------------------------------------------------
+-- Russian/Русский
+----------------------------------------------------------------------
+
+Wire_Keyboard_Remap.Russian = {}
+Wire_Keyboard_Remap.Russian = table.Copy(Wire_Keyboard_Remap_default)
+Wire_Keyboard_Remap.Russian[KEY_RSHIFT] = Wire_Keyboard_Remap.Russian[KEY_LSHIFT]
+
+remap = Wire_Keyboard_Remap.Russian.normal
+remap[KEY_LBRACKET] 	= "х"
+remap[KEY_RBRACKET] 	= "ъ"
+remap[KEY_SEMICOLON] 	= "ж"
+remap[KEY_APOSTROPHE] 	= "э"
+remap[KEY_BACKQUOTE] 	= "ё"
+remap[KEY_COMMA] 		= "б"
+remap[KEY_PERIOD] 		= "ю"
+remap[KEY_SLASH] 		= "."
+remap[KEY_BACKSLASH] 	= "\\"
+remap[KEY_MINUS] 		= "-"
+remap[KEY_EQUAL] 		= "="
+
+local remap = Wire_Keyboard_Remap.Russian.normal
+remap[KEY_NONE] = ""
+remap[KEY_0] = "0"
+remap[KEY_1] = "1"
+remap[KEY_2] = "2"
+remap[KEY_3] = "3"
+remap[KEY_4] = "4"
+remap[KEY_5] = "5"
+remap[KEY_6] = "6"
+remap[KEY_7] = "7"
+remap[KEY_8] = "8"
+remap[KEY_9] = "9"
```

Comment:
You don't have to add `KEY_NONE` and `KEY_0-9` mappings since they are derived from `Wire_Keyboard_Remap_default`.